lib/deps/docs/processDocs.js

```diff
@@ -84,6 +84,9 @@ function processDocs(docInfos, api, fetchedDocs, tx, results, writeDoc, opts,
         updateDoc(fetchedDocs.get(id), currentDoc, results,
           resultsIdx, docWritten, writeDoc, newEdits);
       } else {
+        // Ensure stemming applies to new writes as well
+        var merged =  merge.merge([], currentDoc.metadata.rev_tree[0], 1000);
```

Comment:
Yeh I was planning to do https://github.com/pouchdb/pouchdb/issues/2839 at some point soon